src/App.js

```diff
@@ -1,7 +1,9 @@
 import React from 'react';
 import './App.css';
+import { ProductList } from './components/ProductList';
+import { Button } from './components/Button';
 
-const goodsFromServer = [
+const productFromServer = [
```

Comment:
```suggestion
const productsFromServer = [
```
В тебе ж продуктИ,  а не продукт